addons/binding/org.openhab.binding.osramlightify/src/main/java/org/openhab/binding/lightify/internal/LightifyConnector.java

```diff
@@ -0,0 +1,477 @@
+                                                }
+                                            }
+                                        }
+
+                                        long now = System.nanoTime();
+                                        nextPoll = now + pollInterval;
+                                        if (doDiscovery) {
+                                            doDiscovery = false;
+                                            nextDiscovery = now + bridgeHandler.getConfiguration().discoveryIntervalNanos;
+                                        }
+                                    }
+
+                                    resendCount = 0;
+
+                                    if (initialPoll == null) {
+                                        transmitQueue.sendNext();
+                                    } else {
+                                        initialPoll = null;
+                                        reconnectAttempts = 1;
+
+                                        // Make sure the transmit queue is started. If the queue is empty
+                                        // it will be started when the next request is queued but if we
+                                        // took a connection drop while trying to send a request the queue
+                                        // will NOT be empty and sending will NOT be triggered.
+                                        transmitQueue.send();
+
+                                        // Log the gateway firmware at the earliest opportunity.
+                                        bridgeHandler.sendMessage(new LightifyGatewayFirmwareMessage());
+
+                                        bridgeHandler.setStatusOnline();
+                                    }
+                                } else if (initialPoll != null) {
+                                    disconnect();
+                                } else {
+                                    if (resendCount++ == MAXIMUM_RESENDS) {
+                                        if (request instanceof LightifyListPairedDevicesMessage) {
+                                            havePoll = false;
+                                            pollInterval = bridgeHandler.getConfiguration().minPollIntervalNanos;
+                                            nextPoll = System.nanoTime() + pollInterval;
+                                        }
+
+                                        resendCount = 0;
+                                        transmitQueue.sendNext();
+                                    } else {
+                                        transmitQueue.send();
+                                    }
+                                }
+                            } catch (LightifyException e) {
+                                logger.debug("RX: {}", DatatypeConverter.printHexBinary(messageBuffer.array()));
+                                logger.warn("Error", e);
```

Comment:
Simply add e to the debug log statement, without adding it to the format message (no `{}`).